Formula/bareos-client.rb

```diff
@@ -39,10 +39,14 @@ class BareosClient < Formula
   conflicts_with "bacula-fd", because: "both install a `bconsole` executable"
 
   def install
-    # Work around Linux build failure by disabling warning:
+    # Work around Linux build failure by disabling warnings:
     # lmdb/mdb.c:2282:13: error: variable 'rc' set but not used [-Werror=unused-but-set-variable]
+    # fastlzlib.c:512:63: error: unused parameter ‘output_length’ [-Werror=unused-parameter]
     # TODO: Try to remove in the next release which has various compiler warning changes
```

Comment:
maybe add this ticket info (https://bugs.bareos.org/view.php?id=1504) into the source code?